README.md

```diff
@@ -1 +1,34 @@
-# mobile_menu_training
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Mobile Menu Training
+
+- Here with the use of a provided template we designed a mobile menu for review training
+
+
+## Built With
+
+- HTML, CSS, JavaScript
+
+
```

Comment:
Your Readme is missing the live link demo, it will be better if you add it so other people like recruiters can see what you built